tmk_core/common/keyboard.c

```diff
@@ -383,6 +393,10 @@ void keyboard_task(void) {
     rgblight_task();
 #endif
 
+#ifdef RGB_MATRIX_ENABLE
+    rgb_matrix_task();
```

Comment:
(probably should move the init call to keyboard.c too, for consistency)

Reply:
solid point

Reply:
fixed